src/Properties.cpp

```diff
@@ -17,7 +17,7 @@ int Properties::BAUD_RATE = 9600;
 
 // ********** MODULES ********** //
 boolean Properties::MODULE_WIFI = false;
-boolean Properties::MODULE_PI = false;
+boolean Properties::MODULE_PI = true;
```

Comment:
My bad... I remember thinking about that when I changed it too but I guess my mind was somewhere else when staging files for commit. Thanks for catching it.